sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -53,12 +56,14 @@
     'Top',
     'ToDict',
     'ToList',
+    'Latest'
     ]
 
 # Type variables
 T = TypeVariable('T')
 K = TypeVariable('K')
 V = TypeVariable('V')
+TimestampType = Union[int, long, float, Timestamp, Duration]
```

Comment:
Would not this line fail in python 3 because long is not defined?

/cc @tvalentyn

Reply:
It would, but `from past.builtins import long` on line 31 makes this code py3-compatible.